docs/v2-system-addon/data_events.md

```diff
@@ -1009,3 +1010,19 @@ This reports when an error has occurred when parsing/evaluating a JEXL targeting
   "value": ["MALFORMED_EXPRESSION" | "OTHER_ERROR"]
 }
 ```
+
+### Remote Settings error pings
+
+This reports a failure in the Remote Settings loader to load messages for Activity Stream Router.
+
+```js
+{
+  "action": "asrouter_undesired_event",
+  "client_id": "26288a14-5cc4-d14f-ae0a-bb01ef45be9c",
+  "addon_version": "20180710100040",
+  "locale": "en-US",
+  "user_prefs": 7,
+  "event": ["ASR_RS_NO_MESSAGES" | "ASR_RS_ERROR"],
+  "value": -1
```

Comment:
Nit: for AS Router undesired event, the value could be any of following:
* A String
* An object with key/value pairs
* An array
* Omitted if the value is not applicable for the event